### PR TITLE
Add message about statics figs for mobile devices

### PR DESCRIPTION
**Pull request by brifemu**
In this PR I had added a message just for mobile device users about figs situation on science blog page. Here are some screenshots:

### **EN**

image

### **DE**

image

### CODE REVIEW

File: src/data/science.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
       "url": "/en/science",
       "external": false
     },
+    "mobileMessage": "This page had been optimized for mobile devices, i.e. images as shown as statics figures.",
```
Comment (Ein-Tim) [inline src/data/science.json:15]:
```suggestion
    "mobileMessage": "This page has been optimized for mobile devices, i.e. graphics are shown as statics figures.",
```

**Review by Ein-Tim — COMMENTED** [review]
Suggestion to correct & improve the English text.

File: src/data/science.json (hunk 1 above)
Comment (brifemu) [reply src/data/science.json:15]: What do you think about: ... i.e. graphics are shown as static images. ?

Comment (Ein-Tim) [reply src/data/science.json:15]: Yep, that's better @brifemu!

Comment (brifemu) [reply src/data/science.json]: Thanks, edited and pushed!

File: src/data/science.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
       "url": "/en/science",
       "external": false
     },
+    "mobileMessage": "This page has been optimized for mobile devices, i.e. graphics are shown as static images.",
```
Comment (MikeMcC399) [inline src/data/science.json:15]:
```suggestion
    "mobileMessage": "This page is optimized for mobile devices, i.e. graphics are shown as static images.",
```

Use present tense "is" as the user is interested in the current state of the page, not the history of the change.

File: src/data/science.json [hunk 3]
```diff
@@ -30,6 +31,7 @@
       "url": "/de/science",
       "external": false
     },
+    "mobileMessage": "Die Seite ist für Mobilgeräte optimiert, d.&nbsp;h. Grafiken werden als statische Bilder angezeigt.",
```
Comment (MikeMcC399) [inline src/data/science.json:34]: Maybe "Diese Seite" instead of "Die Seite"?
@Ein-Tim 
What do you think as native German speaker?

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Please check the comments for the English and German texts.

File: src/data/science.json (hunk 3 above)
Comment (Ein-Tim) [reply src/data/science.json:34]: Yes, "Diese" is better 👍

**Review by dsarkar — APPROVED** [review]

**Comment by MikeMcC399** [comment]
@dsarkar 
What about the suggestion to change "Die Seite" to "Diese Seite"?
Do you want this change or will you stay with the current wording?

**Comment by dsarkar** [comment]
@MikeMcC399 "Diese" is better. Thanks.

File: src/data/science.json (hunk 3 above)
Comment (dsarkar) [reply src/data/science.json]: Thanks.

**Comment by dsarkar** [comment]
@MikeMcC399 @Ein-Tim Thanks for review!

**Pull request merged**